repository/repository-core/src/main/java/org/eclipse/vorto/repository/account/impl/DefaultUserAccountService.java

```diff
@@ -159,20 +160,25 @@ public boolean hasRole(String tenantId, String username, String role) {
   }
 
   @Transactional
-  public User create(String username) {
+  public User create(String username, AuthenticationProvider provider, String subject) {
     if (userRepository.findByUsername(username) != null) {
       throw new IllegalArgumentException("User with given username already exists");
     }
 
-    return userRepository.save(User.create(username));
+    return userRepository.save(User.create(username, provider, subject));
   }
 
   @Transactional
-  public User create(String username, String tenantId, Role... userRoles)
+  public User create(String username, String tenantId, Role... userRoles) {
+    return create(username, AuthenticationProvider.GITHUB, null, tenantId, userRoles);
```

Comment:
Github is the default because most of the legacy code calling this method hails from the time when github is our only verification provider.

IMO, this interface method can be deleted, but I'd rather have that as a separate issue.

Reply:
I would not make GitHub as default, because we cannot assume that. I would refactor this in this issue and get rid of this old method. Better clean up now than later :)